vm/src/stdlib/os.rs

```diff
@@ -862,6 +863,22 @@ pub fn os_openpty(vm: &VirtualMachine) -> PyResult {
     }
 }
 
+#[cfg(unix)]
+pub fn os_ttyname(fd: PyIntRef, vm: &VirtualMachine) -> PyResult {
+    use libc::ttyname;
+    unsafe {
```

Comment:
Why is this `unsafe` block required? In any case, please try to reduce the scope of this `unsafe` block. Probably only the `libc` related call is unsafe, the other code is safe rust code.

Reply:
Yes, I see. I can fix it.